scripts/standard_configure.bat

```diff
@@ -16,6 +16,7 @@ rem Set variables
 set KRATOS_SOURCE=~0,-1%/..
 set KRATOS_BUILD=%KRATOS_SOURCE%/build
 set KRATOS_APP_DIR=applications
+rem set KRATOS_INSTALL_PYTHON_USING_LINKS=ON
```

Comment:
I thought thats not working on Win?

Reply:
Apparently it works if you use an elevated prompt

Reply:
@AlejandroCornejo  managed to do it 
https://github.com/KratosMultiphysics/Kratos/issues/6002#issuecomment-566604861
But only with admin permissions
https://github.com/KratosMultiphysics/Kratos/issues/6002#issuecomment-566624369

Reply:
Then I think it makes sense to add a comment telling this

Reply:
Done